test/controllers/notifications_controller_test.rb

```diff
@@ -12,8 +12,8 @@ class NotificationsControllerTest < ActionController::TestCase
   end
 
   test "should assign unapproved sign slides" do
-    users(:two).signs << signs(:one)
-    signs(:one).slides << slides(:two)
+    users(:two).signs << signs(:default)
```

Comment:
Let's get rid of the generic users here, too, then. I'd recommend losing [one, two, james, and ross](https://github.com/chapmanu/signage/blob/43c9c9386ebd6a28300c2aac5501d8702306cd54/test/fixtures/users.yml) and using in their place one of the remaining users.